src/test/rpc/AccountInfo_test.cpp

```diff
@@ -89,11 +89,12 @@ class AccountInfo_test : public beast::unit_test::suite
         Account const alice{"alice"};
         env.fund(XRP(1000), alice);
 
-        auto const withoutSigners =
-            std::string("{ ") + "\"account\": \"" + alice.human() + "\"}";
+        auto const withoutSigners = std::string("{ ") +
+            "\"api_version\": 1, \"account\": \"" + alice.human() + "\"}";
```

Comment:
I may be wrong here but it seems like `api_version` support is broken somehow. 
I'd expect `v1` to be used when no version is specified (as the comment suggests in `RPCHelpers.h`). In turn, if it would use the right version, the test should not be breaking and you do not need to specify `api_version` here and below.

Reply:
For your second and third question, we actually had a bunch of conversation in this [PR](https://github.com/XRPLF/rippled/pull/4552) and elsewhere. I actually wrote a document summarizing the discussion. I'll send you over.

But I added api_version 1 in this test since it was defaulting to version 2 (maxSupportedVersion) which led it to fail. I have described why this happens in the document as well.

With the current state, all api version 1 tests would have to be explicitly set to api_version 1. Reason why I kept this PR to be a draft, because that is a very bad solution. So if we release api version 3,4,5 those will become the default.

However, it is important to note - this is just happening in tests. In reality, the behaviour is opposite. Default behaviours should be api version 1. This is how it works outside of the tests. For example: we need to specify api_version: 2 in the payload if you want to use v2. But v1 is default.

Reply:
But for the first comment, yeah - I stopped working on this after a while due to other stuff. And also because I am not sure what's a good solution to this.

Reply:
Okay, just to clarify here's @ximinez 's comment on a similar question about default API version for command line: https://github.com/XRPLF/rippled/pull/4759#issuecomment-1771618723

It appears that we are defaulting to `RPC::apiMaximumSupportedVersion` (stealing Ed's [example](https://github.com/XRPLF/rippled/blob/89780c8e4fd4d140fcb912cf2d0c01c1b260539e/src/ripple/net/impl/RPCCall.cpp#L1479C36-L1479C67))

So, irrespective of unit-tests or real-world RPC calls, the API version defaults to `RPC::apiMaximumSupportedVersion`

Reply:
No, commandline and unit tests uses max version by default but RPC from my understanding uses the default version which is `apiVersionIfUnspecified` which is currently 1

Reply:
okay, it appears that the above comment holds true for command line usage and not general RPC calls. It has been detailed in the third paragraph of this [document](https://docs.google.com/document/d/1qEMoDyi7or-7ODEfVtnaFlcFSb0NsE2vnvEw6MIJ3Zo/edit?pli=1)

Reply:
that's right

Reply:
ok, thanks for the clarification 👍